webapp/tests/karma/ts/modules/about/about.component.spec.ts

```diff
@@ -120,6 +120,12 @@ describe('About Component', () => {
     expect(spySubscriptionsUnsubscribe.callCount).to.equal(1);
   });
 
+  it('handles missing partners resource - #7100', async(async () => {
```

Comment:
Can you please add a test where it's catching and printing the 404 error? Thanks!

Reply:
Done!